.github/workflows/README.md

```diff
@@ -35,27 +68,24 @@ The CD Workflow is responsible for automatically deploying new code changes to d
 
 
 
-**Workflow Trigger and Jobs**
-
-The CD Workflow is triggered by manual trigger through workflow dispatch, allowing developers to initiate the deployment process on demand.
+**Version Code**
 
-The CD Workflow consists of several jobs, each responsible for a specific deployment task. For instance, the `deploy-to-dev` job deploys the dev build to Firebase, while the `promote-artifact` job promotes the release build to the specified Google Play track.
+The version code is generated from the sum of 2 things:
+1. The unix time in seconds / 1000. This ensures a code that is unique to that moment and always incrementing. 
```

Comment:
You're referring to `unix time in seconds / 1000`? IE The current implementation? 

Did you read the overview? Is your preference option 2?

Reply:
Are we already doing that? :O 

Do we need to truncate it in the first place? maybe we could at least do some more meaningful division - 3600 (per hour) or 900 (per 15 minutes)

Reply:
Yep the current version code is derived from `unix time in seconds / 1000`. For example the currently deployed version code is `1732785`. 

It used to be the bitbucket pipeline number. My understanding is that when we moved to github, github actions at the time did not have the equivalent ability to have a sequentially increasing run number, so it was decided to use a unix timestamp. Which while it works, isn't super useful. 

Now thankfully github actions does have the same feature. Hence my proposal in option 2, because at least this way the version code would mean something, we could take it and point to the specific run that built the deployed release. 

And yes, if we kept option 1 we could truncate it differently! Which it will note down. Though it would always be limited in usefulness. Ex: we can't take `1732785` x 1000 and know exactly when it was built haha.